google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -566,6 +566,48 @@
     <method>java.util.List getFloat32Array()</method>
   </difference>
 
+  <!-- UUID -->
+  <difference>
+    <differenceType>7013</differenceType>
```

Comment:
is 7013 mapped to AbstractStructReader or Value ? 

<difference>
    <differenceType>7013</differenceType>
    <className>com/google/cloud/spanner/AbstractStructReader</className>
    <method>java.util.List getUuidListInternal(int)</method>
  </difference>
  <difference>
    <differenceType>7013</differenceType>
    <className>com/google/cloud/spanner/Value</className>
    <method>java.util.List getUuidArray()</method>
  </difference>

Reply:
7013 is mapped to " (Abstract Method Added to Class): className, method". 
For more mapping: https://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html